zkevm-circuits/src/pi_circuit.rs

```diff
@@ -1036,14 +1477,14 @@ mod pi_circuit_test {
         const MAX_CALLDATA: usize = 8;
         let public_data = PublicData::default();
 
-        let k = 13;
+        let k = 17;
         assert_eq!(run::<Fr, MAX_TXS, MAX_CALLDATA>(k, public_data), Ok(()));
     }
 
     #[test]
     fn test_simple_pi() {
-        const MAX_TXS: usize = 4;
-        const MAX_CALLDATA: usize = 20;
+        const MAX_TXS: usize = 8;
+        const MAX_CALLDATA: usize = 200;
```

Comment:
a small question: why increase these two values and k?   
(bigger k needs longer CI time, so usually we tends to use smaller cases unless needed otherwise)

@kunxian-xia

Reply:
oh it is due to u16 lookup table